lms/static/js/commerce/views/receipt_view.js

```diff
@@ -164,7 +164,7 @@ var edx = edx || {};
          * @return {object} JQuery Promise.
          */
         getCourseData: function (courseId) {
-            var courseDetailUrl = '/api/course_structure/v0/courses/%s/';
+            var courseDetailUrl = '/api/courses/v1/courses/%s/';
```

Comment:
@mattdrayer what about this? A username in the querystring is required for non-anonymous access: https://github.com/edx/edx-platform/commit/8e3f4e058dfc7dd47efde0e5d5637c911fa7cf21#diff-c30da934a615c1eb77a52e6a3f2429baR27.